app/src/main/java/com/ispc/gymapp/options/Mi_Perfil.java

```diff
@@ -0,0 +1,23 @@
+package com.ispc.gymapp.options;
```

Comment:
```suggestion
package com.ispc.gymapp.views.activities;
```
al ser una activity puede ir en el paquete de activities si queres